tests/ui/type/pattern_types/overflowing_literals_in_patterns.rs

```diff
@@ -0,0 +1,18 @@
+//! Check that overflowing literals are rejected just
```

Comment:
They’re not rejected in this test though, right?

Reply:
yea I just fixed it for match patterns not pattern types, as that requires more work... while working on pattern types I noticed it and while fixing it I noticed I bricked it for normal patterns